tests/integration/test_integration_metobs.py

```diff
@@ -3,16 +3,45 @@
 import time
 import pytest
 import datetime
+import urllib
 from smhi.metobs import Metobs, Parameters, Stations, Periods, Data
 
+
+def directdateread(parameter, station, period):
+    """Help function to read date from web directly.
+
+    Args:
+    parameter
+    station
+    period
+    """
+    tmpdata = (
+        "https://opendata-download-metobs.smhi.se/api/version/latest/parameter/"
+        + str(parameter)
+        + "/station/"
+        + str(station)
+        + "/period/"
+        + period
+        + "/data.csv"
+    )
+    a = 0
+    for line in urllib.request.urlopen(tmpdata):
+        a = a + 1
+        if a == 8:
+            tmpstr = line.decode("utf-8")
+            break
+
+    return tmpstr[20:39]
+
+
 METOBS_INTEGRATION = {}
 for i in [1, 2, 22]:
     with open(f"tests/fixtures/metobs_integration_{i}.json", encoding="utf-8") as f:
         METOBS_INTEGRATION[i] = json.load(f)
 
         if i == 1:
-            METOBS_INTEGRATION[i]["Tidsperiod (t.o.m)"] = (
-                datetime.date.today().strftime("%Y-%m") + "-01 07:20:09"
+            METOBS_INTEGRATION[i]["Tidsperiod (t.o.m)"] = directdateread(
```

Comment:
Is this generalisation (and extra logic) needed?

It looks to me like all failing tests are related to station `192840` which has changed `Tidsperiod`, for some reason? We could probably simplify the tests by removing many of these flaky asserts.

Reply:
I mean, this was a mistake from my side when creating the test. The code still works fine even if the test fail because of the date. That is, our tests should fail if something fails (read format changes or something we can't parse) and not if a date changes.

To simplify this, we could simply remove some asserts.

Reply:
Agree. Will simplify and remove some stuff tomorrow. Hanging with wife now 🥰